src/core/client/admin/components/UserRole/UserRoleChange.tsx

```diff
@@ -66,7 +71,8 @@ const UserRoleChange: FunctionComponent<Props> = ({
    */
   const handleChangeRole = useCallback(
     async (r: GQLUSER_ROLE_RL, siteIDs: string[] = []) => {
-      await onChangeRole(r, siteIDs);
+      console.log("changing role, scoped?", !!siteIDs.length);
```

Comment:
would you mind removing this errant console log?

Reply:
✅